config/config_mgmt.py

```diff
@@ -712,18 +715,33 @@ def _deleteHandler(diff, inp, outp, config):
             '''
             if isinstance(inp, dict):
                 # Example Case: diff = PORT': {delete: {u'Ethernet1': {...}}}}
+                self.sysLog(logLevel=syslog.LOG_DEBUG, \
+                    msg="Delete Dict diff:{}".format(diff))
                 for key in diff:
                     # make sure keys from diff are present in inp but not in outp
                     if key in inp and key not in outp:
-                        # assign key to None(null), redis will delete entire key
-                        config[key] = None
+                        if type(inp[key]) == list:
+                            self.sysLog(logLevel=syslog.LOG_DEBUG, \
+                                msg="Delete List key:{}".format(key))
+                            # assign current lists as empty.
+                            config[key] = []
+                        else:
+                            self.sysLog(logLevel=syslog.LOG_DEBUG, \
+                                msg="Delete Dict key:{}".format(key))
+                            # assign current lists as empty.
+                            config[key] = []
```

Comment:
>config[key] = [] [](start = 28, length = 16)

You assign it again in below line.

Reply:
Thanks for finding this, seems by mistake that line 731 got pasted here. And tests will not fail because of "config[key] = None" later.